fenix/app/src/main/res/values/strings.xml

```diff
@@ -1810,14 +1822,24 @@
 
     <!-- Text for the button to create a custom search engine on the Add search engine screen -->
     <string name="search_add_custom_engine_label_other">Other</string>
+    <!-- Label for the TextField in which user enters custom search engine name -->
+    <string name="search_add_custom_engine_name_label" tools:ignore="UnusedResources">Name</string>
     <!-- Placeholder text shown in the Search Engine Name TextField before a user enters text -->
     <string name="search_add_custom_engine_name_hint">Name</string>
+    <!-- Placeholder text shown in the Search Engine Name TextField before a user enters text -->
```

Comment:
```suggestion
    <!-- Placeholder text shown in the Search Engine Name text field before a user enters text -->
```
Let's move away with the technical Android term here even though it already exists in other places. I am sure our translators are just reading it as "text field" anyways.